Sources/ApodiniDatabase/Handlers/Update.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+import Fluent
+@_implementationOnly import Vapor
+import Apodini
+
+/// A Handler that updates a `DatabaseModel` with the given id with the new object of the request.
+/// It uses the database that has been specified in the `DatabaseConfiguration`.
+public struct Update<Model: DatabaseModel>: Handler {
+    @Apodini.Environment(\.database)
+    private var database: Fluent.Database
+    
+    @Parameter
```

Comment:
I think updating would be an other great use cases of `@Properties` and a dynamic creation of these properties. E.g. we could achieve an `Update` component that can update a `Model` with a request that just provides one property of the `Model` and leaves all other elements like they are. I think this would be a very sophisticated usage of reflection and the runtime information and `@Properties` but would showcase the great flexibility of our approaches. I would be fine with this not being part of this PR but a subsequent PR. If this is the case pleas document the feature using a GitHub issue.